docs/sphinx/user_guide/feature/view.rst

```diff
@@ -330,6 +330,60 @@ indices by :math:`N`, ::
     }
   }
 
+Index Layout
+^^^^^^^^^^^^
+
+``RAJA::IndexLayout`` is a layout that uses an indexing strategy for each
+dimension to determine how to map a given input index to an entry within a view.
+
+Three indexing strategies are natively supported in RAJA:
+DirectIndex, IndexList, and ConditionalIndexList. DirectIndex maps an input
+index to itself, and does not take any arguments in its constructor.  The
+IndexList strategy takes a reference to an array containing the list of relevant
+indices to map.  With this strategy, a given input index is mapped to the entry
+in its list corresponding to that index.  Lastly, the ConditionalIndexStrategy
+takes a reference to an array containing an index list.  When the referenced
+index list is non-null, the ConditionalIndex strategy is equivalent to that

```

Comment:
I'm not sure I'm using the right language here.  Is it proper to say "a reference to an array" to mean a reference to the first element of the array?  Also, is it proper to say that a reference to an array is null or not-null?

Reply:
It takes a pointer right? You can say it takes a pointer to an array of indices. I would avoid using reference unless you're actually talking about references in a c++ language sense (`int&`).

Reply:
@gberg617 FYI, in valid C++ code, there is no way to create a null reference. Each reference must be assigned to a valid object where the reference is defined. It is possible to encounter a "null" reference in the sense that it does not refer to a valid object, but that means something is wrong with the code.